scripts/scripts.js

```diff
@@ -673,12 +674,41 @@ function loadFooter(footer) {
 function buildAutoBlocks(main) {
   try {
     buildHeroBlock(main);
+    if (getMetadata('show-banner')) {
```

Comment:
I would make this slightly smarter to allow authors to enable or disable it:
```suggestion
    if (['yes','on'].includes(getMetadata('show-banner'))) {
```